src/libraries/src.proj

```diff
@@ -1,5 +1,9 @@
-<Project DefaultTargets="Build" TreatAsLocalProperty="AdditionalBuildTargetFrameworks">
-  <Import Project="$(MSBuildThisFileDirectory)Directory.Build.props" />
+<Project InitialTargets="IncludeProjectReferences">
+  <Import Sdk="Microsoft.NET.Sdk" Project="Sdk.props" />
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp5.0</TargetFramework>
```

Comment:
$(NetCoreAppCurrent) instead?

Reply:
This one is problematic. I believe we need to set a TargetFramework here so that NuGet doesn't complain. But I'm not sure if this TargetFramework has any impact on the chosen projects. I don't know what the right way is here, maybe setting it to BuildTargetFramework? cc @Anipik 

This needs to be solved for ref.proj, src.proj and tests.proj.